examples/AcousticWave2D_codegen.py

```diff
@@ -78,14 +78,21 @@ def damp_boundary(damp):
         self.dm = dm
 
     def Forward(self):
-        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True, auto_tune=True)
+        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True)
         fw.apply()
-        return (self.rec.data, self.u.data)
+        return self.rec.data, self.u.data
+
+    # Forward with auto tuning enabled
+    def Forward_auto_tune(self):
```

Comment:
Why are we adding this function and a whole new example file if we could simply propagate one constructor variable? We're not running this as part of the travis test runs, so why the code duplication?